server/methods/accounts/accounts.app-test.js

```diff
@@ -56,100 +68,90 @@ describe("Account Meteor method ", function () {
   function spyOnMethod(method, id) {
     return sandbox.stub(Meteor.server.method_handlers, `cart/${method}`, function (...args) {
       check(args, [Match.Any]); // to prevent audit_arguments from complaining
-      this.userId = id;
+      this.userId = id; // having to do this makes me think that we should be using Meteor.userId() instead of this.userId in our Meteor methods
       return originals[method].apply(this, args);
     });
   }
 
   describe("addressBookAdd", function () {
     beforeEach(function () {
-      Cart.remove({});
-      Accounts.remove({});
+      const sessionId = Random.id(); // Required for creating a cart
+      // editing your address book also udpates your cart, so be sure there
+      // is a cart present
+      Meteor.call("cart/createCart", fakeAccount.userId, sessionId);
     });
 
-    it("should allow user to add new addresses", function (done) {
-      let account = Factory.create("account");
```

Comment:
previously, we had created the user once at the top of the file (i.e., not in a beforeEach). In doing so, creating an account here made sense. the beforeEach now creates a user and an account to cut down on some of the boiler plate test code